docs/topics/clusterdefinitions.md

```diff
@@ -649,12 +649,74 @@ https://{keyvaultname}.vault.azure.net:443/secrets/{secretName}/{version}
 | imageVersion                     | no       | Specific image version to deploy from marketplace.  Default: `17763.615.1907121548`. This default is incremented as new versions are tested to avoid unexpected breaks. |
 | windowsImageSourceURL            | no       | Path to an existing Azure storage blob with a sysprepped VHD. This is used to test pre-release or customized VHD files that you have uploaded to Azure. If provided, the above 4 parameters are ignored. |
+| imageReference.name              | no       | Name of an Image. |
+| imageReference.resourceGroup     | no       | Resource group that contains the Image. |
+| imageReference.subscriptionId    | no       | ID of subscription containing a Shared Image Gallery. |
+| imageReference.gallery           | no       | Name of a Shared Image Gallery. |
+| imageReference.version           | no       | Version of an Image from a Shared Image Gallery. |
 | sshEnabled                       | no       | If set to `true`, OpenSSH will be installed on windows nodes to allow for ssh remoting. **Only for Windows version 1809/2019 or later** . The same SSH authorized public key(s) will be added from [linuxProfile.ssh.publicKeys](#linuxProfile) |
 
+#### Windows Images 
 
-#### Choosing a Windows version
+You can configure the image used for all Windows nodes one of the following ways:
 
-If you want to choose a specific Windows image, but automatically use the latest - set `windowsPublisher`, `windowsOffer`, and `windowsSku`. If you need a specific version, then add `agentWindowsVersion` too.
+##### Custom VHD
+
+To use an image uploaded to an Azure storage account (or any other accessible location) specify `windowsImageSourceURL`.
+
+```json
+"windowsProfile": {
+            "adminUsername": "...",
+            "adminPassword": "...",
+            "windowsImageSourceURL": "https://images.blob.core.windows.net/vhds/custom_windows_image.vhd",
+     },
+```
+
+##### Azure Image
+
+To use a pre-existing Azure Image specify `imageReference.name` and `imageRefernce.ResourceGroup`.
+
+```json
+"windowsProfile": {
+            "adminUsername": "...",
+            "adminPassword": "...",
+            "imageRefernce": {
+              "name": "custom-image",
+              "resourceGroup": "windows-images"
+            }
+     },
+```
+
+##### Shared Image Gallery
+
+To use an Image from a Shared Image Gallery specify `imageReference.name`, `imageReference.ResourceGroup`, `imageReference.subscriptionId`, `imageReference.galllery`, and `imageReference.version`.
+
+```json
+"windowsProfile": {
+            "adminUsername": "...",
+            "adminPassword": "...",
+            "imageRefernce": {
+              "name": "custom-image",
+              "resourceGroup": "windows-images",
+              "subscriptionId": "00000000-0000-0000-0000-000000000000",
```

Comment:
is there any reason tenantId is needed too?

Reply:
I didn't see any fields that could be set for tenant id.